scripts/development/m3_stack/prometheus01.yml

```diff
@@ -0,0 +1,57 @@
+global:
+  # Don't apply external labels for 
+  # external_labels:
+  #   role: "remote"
+  scrape_interval:     10s
+  evaluation_interval: 10s
+
+# Alertmanager configuration
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      # - alertmanager:9093
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+  # - "first_rules.yml"
+  # - "second_rules.yml"
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'prometheus01'
+
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+
+    static_configs:
+      - targets: ['prometheus01:9090']
+
+  - job_name: 'coordinator'
+    static_configs:
+      - targets: ['m3coordinator01:7203']
+
+  - job_name: 'dbnode'
+    static_configs:
+      - targets: ['m3db_seed:9004', 'm3db_data01:9004', 'm3db_data02:9004']
+
+  - job_name: 'aggregator'
+    static_configs:
+      - targets: ['m3aggregator01:6002']
+
+remote_read:
```

Comment:
Can you add `read_recent: true` to the `remote_read` stanza? (Also to the regular prom yml)

Reply:
Sounds good.